react/features/base/conference/actionTypes.js

```diff
@@ -62,6 +62,17 @@ export const CONFERENCE_WILL_LEAVE = Symbol('CONFERENCE_WILL_LEAVE');
  */
 export const DATA_CHANNEL_OPENED = Symbol('DATA_CHANNEL_OPENED');
 
+/**
+ * The type of action which signals that the user has been kicked out from
+ * the conference.
+ *
+ * @type {
```

Comment:
Technically, if we wanted to describe a `@type`, we'd use `{{}}` because the first/outer set of `{}` would be for `@type {}` which is how we're supposed to describe types and the second/inner set of `{}` would be part of the very type description to signify it's an `object`. Anyway, in this case we're not describing the type of the variable `KICKED_OUT` so there's no need for `@type` and, consequently, a single set of `{}` is enough. Indeed, nowhere in this file we have `@type`. I'm rebasing to the latest master to retest so I can very well drop the `@type` myself.

Reply:
sorry that was a copy paste error